src/invitations.ts

```diff
@@ -99,12 +99,11 @@ export default class Invitations {
     }
   }
 
-  public async Invitations (queryParams: PageMetadata, domainId: string, token: string): Promise<InvitationsPage> {
+  public async Invitations (queryParams: PageMetadata, token: string): Promise<InvitationsPage> {
```

Comment:
add back `domainId` as part of the query params. We will need it to list invitations in a particular domain